pkg/riff/commands/docs_test.go

```diff
@@ -82,3 +85,35 @@ func TestDocsCommand(t *testing.T) {
 
 	table.Run(t, commands.NewDocsCommand)
 }
+
+func TestLineWrapping(t *testing.T) {
+	failed := false
+	rootCommand := commands.NewRootCommand(context.Background(), cli.Initialize())
+	err := cli.Visit(rootCommand, func(c *cobra.Command) error {
+		if !isValid(c.Short) {
+			failed = true
+			t.Logf("command `%s`'s Short description exceeds the 80-character margin", c.UseLine())
+		}
+		if !isValid(c.Long) {
+			failed = true
+			t.Logf("command `%s`'s Long description exceeds the 80-character margin", c.UseLine())
+		}
+		return nil
+	})
+	if err != nil {
+		t.Errorf("%v", err)
+	}
+	if failed {
+		t.Errorf("Line wrapping not properly applied by some commands - see above")
+	}
+}
+
+func isValid(doc string) bool {
+	lines := strings.Split(doc, "\n")
+	for _, line := range lines {
+		if len(line) > 80 {
```

Comment:
A number of lines that were exactly 80 characters were unnecessarily wrapped

```suggestion
		if len(line) >= 80 {
```

Reply:
what's the resolution?

Reply:
yep, the initial impl was correct, my doc corrections were not